doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+```
+
+
+### Connections and Layout ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Connect plug                          Click and drag plug to another plug
+Disconnect plug                       Click and drag connection to background
+Connect to bookmarked node            Right-click plug > `Connect Bookmark` > select node
+Insert node at connection             Drag node onto connection
+Auto-arrange selected nodes           :kbd:`Ctrl`-:kbd:`L`
+===================================== =============================================
+```
+
+
+### Navigation ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame selected nodes                  :kbd:`F`
+Enter `Box` node (subgraph)	          :kbd:`↓`
+Leave `Box` node (subgraph)           :kbd:`↑`
+Search for nodes                      :kbd:`Ctrl`-:kbd:`F`
+Bookmark node                         Right-click on node
+===================================== =============================================
+```
+
+
+## _Viewer_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Viewer_ tab.
+
+
+### 2D Images ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut/gesture
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame image                           :kbd:`F`
+Center image at 1:1 scale             :kbd:`Home`
+Toggle red channel                    :kbd:`R`
+Toggle green channel                  :kbd:`G`
+Toggle blue channel                   :kbd:`B`
+Toggle alpha channel                  :kbd:`A`
+===================================== =============================================
+```
+
+### 3D Scenes ###
+
+```eval_rst
+========================================== =====================================
+Action			                           Shortcut/gesture
+========================================== =====================================
+Tumble                                     :kbd:`Alt`-click and drag
+Track                                      :kbd:`Alt`-middle-click and drag
+Dolly                                      :kbd:`Alt`-right-click and drag
```

Comment:
Maybe call this "Dolly / Zoom" to clarify its basically the same as 2d Viewer Zoom?